tools/benchmark/memory/src/main.rs

```diff
@@ -102,7 +102,7 @@ fn main() {
             .arg("--features")
             .arg("icu_benchmark_macros/benchmark_memory")
             .arg("--features")
-            .arg("memory")
+            .arg("bench")
```

Comment:
```suggestion
            .arg("bench-memory")
```

Reply:
`--features bench` is correct and it shouldn't have been changed in my previous commit

Reply:
Ah I see. It would probably make sense to build in `bench-memory` here, then the `cargo run` later will be free. As is we're building everything twice.

Reply:
Good observation. Fixed